spec/node_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'node'
+
+describe Node do
```

Comment:
If you will do a module, probably you have to change the test implementation. You can take a look at this article about how to test Modules in isolation test.

https://chriskottom.com/blog/2015/03/testing-ruby-mixins-in-isolation/